Libs/MRML/Logic/vtkMRMLColorLogic.h

```diff
@@ -125,6 +176,118 @@ class VTK_MRML_LOGIC_EXPORT vtkMRMLColorLogic : public vtkMRMLAbstractLogic
   /// procedural node with a blank color transfer function.
   static vtkMRMLProceduralColorNode* CopyProceduralNode(vtkMRMLColorNode* colorNode, const char* copyName);
 
+  /// Return the label's terminology in this color table.
+  bool LookupCategorizationFromLabel(int label, ColorLabelCategorization&, const char *lutName = "GenericAnatomyColors");
+  /// Returns a string with the terminology term requested, or an empty string if not found.
+  /// Valid values for categorization are SegmentedPropertyCategory, SegmentedPropertyType,
+  /// SegmentedPropertyTypeModifier, AnatomicRegion, AnatomicRegionModifier.
+  /// Valid values for standard term are CodeValue, CodeMeaning, CodingSchemeDesignator
+  /// \sa LookupCategorizationFromLabel
+  std::string GetTerminologyFromLabel(std::string categorization, std::string standardTerm, int label,
+                                      const char *lutName = "GenericAnatomyColors");
+  /// From a given terminology, find the label value that corresponds to it for this color table.
+  bool LookupLabelFromCategorization(ColorLabelCategorization&, int&, const char *lutName = "GenericAnatomyColors");
+  /// Print the terminology for this label in the color table to standard output.
+  bool PrintCategorizationFromLabel(int label, const char *lutName = "GenericAnatomyColors");
```

Comment:
Since you use `"GenericAnatomyColors"` everywhere as a default, may be you could add a 

`const char* DEFAULT_LUT_NAME` to your class and use that instead of a string.

Less chance of doing a typo